src/utils/smartParser.ts

```diff
@@ -191,10 +191,18 @@ export const smartParser = (
                         fileContent[nestedArrayName][categorySelector].length;
                         j++
                       ) {
-                        const parsedDataPoint = getParsedDataPoint(
-                          filePathModel,
-                          nestedArrayName
-                        );
+                        const parsedDataPoint = getEmptyDataPoint();
```

Comment:
removed with the merge ?

Reply:
Je pense que ce n'est pas une bonne idée de retirer cette ligne. J'ai fait le test d'enlever cette variable dans chaque partie de la fonction mais ça m'a sorti pas mal de nouvelles erreurs. J'aimerais bien avoir la confirmation de @JeFaisLeCafe sur l'objectif de cette ligne (si possible mais pas indispensable pour merge).

![Uploading Screenshot 2022-05-06 at 09.35.27.png…]()

Reply:
Ma question était plutôt c’est via feature/redux que ces lignes ont été delete ou c’était initialement comme ça ? @tangigouez

Reply:
Non c'est moi qui a delete ce bloc et remplacé par le code dessous parce que la fonction ne couvrait pas le use case.